File: 240213/최소공배수 구하기/find-the-least-common-multiple.py
n, m = map(int, input().split())

def lcm(a, b):
    for i in range(min(a, b), 0, -1):
        if a % i == 0 and b % i == 0:
            print(a*b//i)
            break

lcm(n, m)

File: 240805/최소공배수 구하기/find-the-least-common-multiple.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ").map(Number);
let n = input[0];
let m = input[1];

function lcm(n, m) {
    let min = Math.min(n, m);
    for (let i = min; i <= n * m; i += min) {
        if (i % n === 0 && i % m === 0) {
            process.stdout.write(`${i}`);
            break;
        }
    }
}

lcm(n, m);